.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      crate:
+        description: Crate to bump (anything which depends on this crate will also be bumped)
+        type: choice
+        default: wdl-grammar
```

Comment:
Would the default of `wdl` make sense?

Reply:
The current default (`wdl-grammar`) is the same as saying "bump everything" which I think is our most common use. Specifying `wdl` would only bump that individual crate because it sits at the top of the dependency tree.

Reply:
Oh right, I was thinking in _publish_ order, so basically the reverse of what we want. Makes sense 👍.